front-dispatcher/src/strings/loginStrings.ts

```diff
@@ -0,0 +1,4 @@
+export const loginString = {
+  title: "Welcome to Dispatcher",
```

Comment:
It's not that bad but it's more common to make a single string file called strings.ts and name it to centralize everything